wildlifelicensing/apps/emails/emails.py

```diff
@@ -69,6 +69,9 @@ def send(self, to_addresses, from_address=None, context=None, attachments=None,
             to_addresses = [to_addresses]
         if attachments is not None and not isinstance(attachments, list):
             attachments = list(attachments)
+        else:
+            attachments = []
+
```

Comment:
I just realized that this is wrong. 
The else should be replaced by:
``` 
if attachments is None:
    attachments = []
```
Currently if we pass a `list` of attachments it get converted to `[]`